src/Engine/GameEvents/ModifyFundsEvent.java

```diff
@@ -1,38 +1,39 @@
 package Engine.GameEvents;
 
-import java.util.function.Consumer;
-
+import CommandingOfficers.Commander;
 import Engine.XYCoord;
 import Terrain.MapMaster;
 import UI.MapView;
 import UI.Art.Animation.GameAnimation;
 
-public class LambdaEvent implements GameEvent
+public class ModifyFundsEvent implements GameEvent
 {
-  private final Consumer<MapMaster> lambda;
+  private final Commander beneficiary;
+  private final int value;
 
-  public LambdaEvent( Consumer<MapMaster> lambda )
+  public ModifyFundsEvent(Commander beneficiary, int value)
   {
-    this.lambda = lambda;
+    this.beneficiary = beneficiary;
+    this.value = value;
   }
 
   @Override
   public GameAnimation getEventAnimation(MapView mapView)
   {
-    // TODO?
+    // TODO
```

Comment:
I'm not sure this needs an animation at all; probably don't need a TODO here.

Reply:
The carts have an animation for it, actually.

Reply:
Just the number-scrolling animation? I mean, sure. That's probably low enough of a priority that I'm not sure we need a TODO anyway.